features/PluginContext.feature

```diff
@@ -32,14 +32,15 @@ Feature: Plugin context
 
   @realtime
   Scenario: Subscribe and unsubscribe to realtime notifications
+    Given a plugin realtime subscription
```

Comment:
This could be replaced by 
```suggestion
    Given I successfully execute the action "functional-test-plugin/realtime":"subscribeOnce"
```

And the associated step can be removed

Reply:
Less is more :+1: 

Changes applied.